client/src/components/pages/common/Home/index.js

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import DotsStepper from '../../../application/DotsStepper';
+import registrationSteps from '../../../application/DotsStepper/registrationSteps.json';
+import draw1 from '../../../../assets/images/draw1.svg';
 
-const Home = () => <h1>Home</h1>;
+import './index.css';
 
+const Home = () => (
+  <>
+    <div className="root">
+      <DotsStepper steps={registrationSteps.registrationSteps} />
```

Comment:
try to edit in DotsStepper Component & reduce distance between 
`TO BE ELIGIBLE FOR THE PROGRAM` sentence 
& Registration steps